src/main/java/com/simibubi/create/content/kinetics/chainConveyor/ChainConveyorBlockEntity.java

```diff
@@ -87,7 +88,7 @@ public ChainConveyorBlockEntity(BlockEntityType<?> typeIn, BlockPos pos, BlockSt
 
 	@Override
 	protected AABB createRenderBoundingBox() {
```

Comment:
While you're here, this could be made much more accurate if the render bounding box is made to enclose all positions that this conveyor has a connection with